From 8ae71268e6de91f7c157e413c0ca90fa64ee009a Mon Sep 17 00:00:00 2001
Message-ID: <8ae71268e6de91f7c157e413c0ca90fa64ee009a.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 01:23:21 +0100
Subject: [PATCH 18/19] gnu: go-golang-org-sql-mock: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-golang-org-sql-mock): Move from
here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 27 +++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 25 -------------------------
 2 files changed, 27 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 56f412224e..8fc9d2fbc0 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -1,6 +1,7 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2018 Pierre Neidhardt <mail@ambrevar.xyz>
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
+;;; Copyright © 2019 Brian Leung <bkleung89@gmail.com>
 ;;; Copyright © 2019 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2020 Jakub Kądziołka <kuba@kadziolka.net>
 ;;; Copyright © 2020 Joseph LaFreniere <joseph@lafreniere.xyz>
@@ -458,6 +459,32 @@ (define-public go-gopkg-in-go-playground-assert-v1
 custom assertions to be used alongside native Go testing.")
     (license license:expat)))
 
+(define-public go-golang-org-sql-mock
+  (let ((commit "e98392b8111b45f8126e00af035a0dd95dc12e8b")
+        (version "1.3.3")
+        (revision "1"))
+    (package
+      (name "go-golang-org-sql-mock")
+      (version (git-version version revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/DATA-DOG/go-sqlmock")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "033vv29g2wf6fd757ajfmha30bqin3b07377037zkl051mk6mghs"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/DATA-DOG/go-sqlmock"))
+      (synopsis "Mock library implementing @code{sql/driver}")
+      (description "This library simulates SQL-driver behavior in tests
+without requiring a real database connection.")
+      (home-page "https://github.com/DATA-DOG/go-sqlmock")
+      (license license:expat))))
+
 (define-public go-golang-org-x-lint
   (let ((commit "83fdc39ff7b56453e3793356bcff3070b9b96445")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index e5dd116766..aedf509c97 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -7228,31 +7228,6 @@ (define-public go-github-com-kr-text
     (home-page "https://github.com/kr/text")
     (license license:expat)))
 
-(define-public go-golang-org-sql-mock
-  (let ((commit "e98392b8111b45f8126e00af035a0dd95dc12e8b")
-        (version "1.3.3")
-        (revision "1"))
-    (package
-      (name "go-golang-org-sql-mock")
-      (version (git-version version revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/DATA-DOG/go-sqlmock")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "033vv29g2wf6fd757ajfmha30bqin3b07377037zkl051mk6mghs"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/DATA-DOG/go-sqlmock"))
-      (synopsis "Mock library implementing @code{sql/driver}")
-      (description "This library simulates SQL-driver behavior in tests
-without requiring a real database connection.")
-      (home-page "https://github.com/DATA-DOG/go-sqlmock")
-      (license license:expat))))
-
 (define-public go-github-com-go-sql-driver-mysql
   (package
     (name "go-github-com-go-sql-driver-mysql")
-- 
2.41.0

